rpi-update

```diff
@@ -54,7 +54,7 @@ else
         BOOT_PATH=${BOOT_PATH:-"/boot"}
 fi
 WORK_PATH=${WORK_PATH:-"${ROOT_PATH}/root"}
-SKIP_KERNEL=${SKIP_KERNEL:-0}
+SKIP_KERNEL=${SKIP_KERNEL:-}
```

Comment:
isn't this now a no-op?

Reply:
Does it work without it?

Reply:
I seem to remember getting some complaint about an uninitialised variable, but that could have been with a version of the line before this.

Reply:
I believe bash variables don't need to be initialised to compare them:
```
$ [ "$NOTSETVARIABLE" == "" ] && echo yes
yes
```

Reply:
```
./rpi-update: line 85: SKIP_KERNEL: unbound variable
```

Reply:
Ah - I didn't realise we did `set -o nounset`. All fine then.